tests/test_cli.py

```diff
@@ -83,6 +83,42 @@ def test_create_study_command_with_direction() -> None:
             subprocess.check_call(command)
 
 
+def test_create_study_command_with_multiple_directions() -> None:
```

Comment:
A minor but how about adding a test where both `--direction` and `--directions` are given?

Reply:
I added the case in 25a67e0. ☝️

Reply:
Thanks!